docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -392,3 +393,18 @@ public static void main(String[] args) throws SQLException {
 
 ## Enable Logging
 To enable logging in the AWS JDBC Driver, change the `logger=StandardLogger` parameter to `wrapperLoggerLevel=FINEST`
+
+
+## Enable Third Party Classes and Packages
+A user application may require a special handling for particular classes or Java packages. In such case, wrapping of data objects by the AWS Advanced JDBC Driver might be crucial for a user application. The driver allows to specify classes and packages that should be left intact with no wrapping. Use the following code snippet as an example to register classes or an entire package if needed.
+
+```java
+Driver.skipWrappingForType(com.pgvector.PGvector.class);
+Driver.skipWrappingForType(com.pgvector.PGhalfvec.class);
+Driver.skipWrappingForType(com.pgvector.PGbit.class);
+    
+Driver.skipWrappingForPackage("com.pgvector");
+```
+This feature can be used to allow the AWS Advanced JDBC Driver to properly handle popular database extensions like [PGvector](https://github.com/pgvector/pgvector-java) and [PostGIS](https://github.com/postgis/postgis-java).
```

Comment:
Curious if we need to expand on the consequences of wrapping these packages? If there are any.